documentation/docs/configuration.md

```diff
@@ -2,6 +2,9 @@
 
 Configuration is done via a yml-file, located at `.pipeline/config.yml` in the **master branch** of your source code repository.
 
+!!! note "Cloud SDK Pipeline"
```

Comment:
I think it is sad to have such kind of different approaches (`pipeline_config.yml`, `.pipeline/config.yml`). Having such kind of notes here is a smell. The dotfile/dotfolder approach is a commonly used standard.

Reply:
I agree, the current state is not good. It hurt writing that down, but as far as I can tell modifying it now while keeping backwards compatibility is not totally trivial.